docs/content/configuration/csi_ug_config_create_secret_topology.md

```diff
@@ -0,0 +1,48 @@
+# Creating a Secret with topology awareness
+
+Create an array secret YAML file in order to define the storage credentials (username and password) and address. Use this information for creating a Secret that is topology aware.
+
+**Note:** If you are not using the CSI Topology feature, follow the steps in [Creating a Secret](csi_ug_config_create_secret.md).
+
+Use the Secret file to connect a worker node to a storage system.
+
+Within the Secret, each `management-id` (here, represented by `demo-management-id-x`), is used to identify the storage system within other configuration files.
```

Comment:
if there was a name for this field, our parameters naming convention actually has underscores (and not hyphens):
```suggestion
Within the Secret, each `management_id` (here, represented by `demo-management-id-x`), is used to identify the storage system within other configuration files.
```
hopefully this would be easier to relate to `by_management_id`

Reply:
@oriyarde  

1. If the convention is to use underscores, then shouldn't `demo-management-id-x` be changed in the example json as well? This is where I took the information from.
2. Upon further inspection of the JSON, perhaps I should write: "Within the Secret, each **`management_address`** (here, represented by xxxx)..."

FYI @ArbelNathan

Reply:
1. we have a different convention for field names and field values. the naming convention comes from python variables snake_case, and IMO the hyphenated value convention helps distinguish it from the name.
2. `management_address` is a different field than `management_id`

Reply:
I see now - 
in any case - as you can see in my markup below - the management-id that I'm referring to (with the clear example) is for the name which, in the json example is also with the hyphens. 

![image](https://user-images.githubusercontent.com/54570654/134345358-ae1c5392-7baa-450e-8c18-5a60c312d4f3.png)

Reply:
I see, then I suggest:
```suggestion
Within the Secret, each management id (here, represented by `demo-management-id-x`), is used to identify the storage system within other configuration files.
```
since there is no hyphenated `management-id` without `demo-` anywhere

Reply:
DONE